ext/curl/curl_string_file.stub.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/** @generate-function-entries */
+
+class CURLStringFile
+{
+    public function __construct(string $data, string $posted_filename, ?string $mime_type = null) {}
+
+    /** @return string */
+    public function getData() {}
+
+    /** @return string */
+    public function getPostFilename() {}
```

Comment:
CURLFile also has a `setPostFilename` method.

Though, taking a step back here, seeing as the class has public properties, these getters and setters are really redundant. I think we'd be better of declaring them as typed properties and just omitting the methods entirely.

It depends on whether we need to have the same interface as CURLFile. It doesn't seem necessary to me, as I would expect the common usage to just construct CURLStringFile and directly pass it to curl.